resources/articles/platform_adhesion/skirt_gap.md

```diff
@@ -2,6 +2,7 @@ Skirt Distance
 ====
 This setting adjusts the distance between the model and the skirt.
 
-By keeping enough distance, the skirt won't attach to the model, reducing the elephant's feet effect.
+This is the distance between the closest line of the skirt and the model. 
+If the skirt consists of multiple lines, these additional lines will be placed further away from the model.
 
-This is the distance of the closest line of the skirt. If the skirt consists of multiple lines, those will be placed further.
\ No newline at end of file
+By keeping enough distance, the skirt won't attach to the model, reducing the elephant's feet effect.
```

Comment:
Not sure whether elephant's feet should be plural or not.
All3DP seems to use [elephant's foot](https://all3dp.com/2/elephant-s-foot-3d-printing-problem-easy-fixes/#:~:text=Sometimes%20elephant's%20foot%20is%20simply,forcing%20it%20to%20bulge%20out.) singular.
Prussa uses: [elephant foot](https://help.prusa3d.com/article/elephant-foot-compensation_114487) (**with no _'s_**)

Reply:
I investigated this when I made the Elephant's Foot troubleshooting article and landed on "elephant's foot" (with 's) being the most common spelling in the industry. So I think we should correct it to become singular.